samples/cellular/https_client/overlay-tfm_mbedtls.conf

```diff
@@ -10,11 +10,14 @@ CONFIG_MBEDTLS_ENABLE_HEAP=y
 CONFIG_MBEDTLS_SSL_IN_CONTENT_LEN=4096
 CONFIG_MBEDTLS_SSL_OUT_CONTENT_LEN=4096
 CONFIG_MBEDTLS_HEAP_SIZE=32768
+CONFIG_MBEDTLS_RSA_C=y
 CONFIG_MBEDTLS_TLS_LIBRARY=y
 CONFIG_MBEDTLS_X509_LIBRARY=y
 CONFIG_MBEDTLS_PKCS1_V15=y
 CONFIG_NRF_SECURITY_ADVANCED=y
 CONFIG_NORDIC_SECURITY_BACKEND=y
+CONFIG_PSA_CRYPTO_DRIVER_CC3XX=y
```

Comment:
Can you open a bit why this is enabled? 

It seems to have some configuration interaction with CONFIG_BUILD_WITH_TFM, which, if I recall, is enabled in this sample.

Reply:
It is to enable the [Arm CryptoCell cc3xx driver](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/libraries/nrf_security/doc/drivers.html#id7) for hardware-accelerated cryptography using the Arm CryptoCell cc3xx hardware. It is not strictly needed.